packages/console/src/components/Card/index.tsx

```diff
@@ -1,15 +1,22 @@
 import classNames from 'classnames';
-import React, { ReactNode } from 'react';
+import React, { forwardRef, ReactNode } from 'react';
 
 import * as styles from './index.module.scss';
 
 type Props = {
   children: ReactNode;
+  // eslint-disable-next-line react/require-default-props
```

Comment:
If I recall properly there is a workaround. Not sure if it works, but could give it a try

```js
const Card =((props: Props, reference: $SomeType ) => {
  const { children, className } = props;

  return (
    <div ref={reference} className={classNames(styles.card, className)}>
      {children}
    </div>
  );
});

export default forwardRef<HTMLDivElement, Props>(Card);
```

Reply:
Yeah, this works. Nice!